frontend/src/app/modules/leader/leader-board/leader-board.component.sass

```diff
@@ -0,0 +1,64 @@
+@import 'color-variables'
+
+.wrapper
+    margin-top: 30px
+    display: flex
+    justify-content: center
+    align-items: center
+    height: 100%
+
+.board
+    width: 100%
+
+.board-wrapper
+    display: flex
+    background-color: $midnight-gray
+    margin: auto
+    
+
+.header
+    gap: 16px
+    margin-left:8px
+    div
+        padding-bottom: 11px
+        padding-top: 11px
+        color: $light-blue-grey
+        font-family: Inter
+        font-size: 12px
+        font-style: normal
+        font-weight: 400
+        line-height: 21px
```

Comment:
i guess font-style: normal; font-weight: 400 can be removed everywhere without any effect